doc/builders/trivial-builders.chapter.md

```diff
@@ -31,34 +46,53 @@ An example of using `runCommand` is provided below.
 
 ## `runCommandCC` {#trivial-builder-runCommandCC}
 
-This works just like `runCommand`. The only difference is that it also provides a C compiler in `buildCommand`'s environment. To minimize your dependencies, you should only use this if you are sure you will need a C compiler as part of running your command.
+This works just like `runCommand`. The only difference is that it also provides a C compiler in `buildCommand`'s environment.
+To minimize your dependencies, you should only use this if you are sure you will need a C compiler to run your command.
 
 ## `runCommandLocal` {#trivial-builder-runCommandLocal}
 
-Variant of `runCommand` that forces the derivation to be built locally, it is not substituted. This is intended for very cheap commands (<1s execution time). It saves on the network round-trip and can speed up a build.
+A variant of `runCommand` that ensures the derivation is built locally instead of substituting a remote build.
+It can save time and speed up a larger collection of builds by avoiding the network round-trip to check if a remote substitution is available.
+However, it is only intended for very cheap commands (<1s execution time).
 
 ::: {.note}
-This sets [`allowSubstitutes` to `false`](https://nixos.org/nix/manual/#adv-attr-allowSubstitutes), so only use `runCommandLocal` if you are certain the user will always have a builder for the `system` of the derivation. This should be true for most trivial use cases (e.g., just copying some files to a different location or adding symlinks) because there the `system` is usually the same as `builtins.currentSystem`.
+This sets `allowSubstitutes` to `false`.
+Only use `runCommandLocal` if you are certain the user will always have a builder for the `system` of the derivation.
+This should be true for most trivial use cases (e.g., just copying some files to a different location or adding symlinks)
+because there the `system` is usually the same as `builtins.currentSystem`.
```

Comment:
Let's link to https://nixos.org/manual/nix/stable/command-ref/conf-file.html?highlight=nix.conf#conf-system instead, documents what this is